_includes/scripts/jqueryui.js

```diff
@@ -23,14 +23,15 @@ $(document).on('click','a[data-toggle="datepicker"]', function(e){
 //
 // Autocomplete
 // --------------------------------------------------
-
-$.extend($.ui.autocomplete.prototype.options, {
-	open: function(event, ui) {
-		$(this).autocomplete("widget").css({
-			"width": ($(this).outerWidth() + "px")
-		});
-	}
-});
+if ($.ui.autocomplete) {
```

Comment:
dbt Cloud can't resolve `$.ui.autocomplete`. I haven't found usage of ui.autocomplete in dbt Cloud, so I think this is a safe change.